doc/xl2tpd.8

```diff
@@ -1,70 +1,89 @@
-.TH "xl2tpd" "8" "" "Jeff McAdams" ""
-.SH "NAME"
-xl2tpd \- Layer 2 Tunnelling Protocol Daemon
-.SH "DESCRIPTION"
-A Layer 2 Tunneling Protocol VPN client/daemon for Linux and other POSIX-based
-OSs. Based off of L2TPd 0.61 from 
-  http://www.marko.net/l2tp 
-and patches collected by Jacco de Leeuw at
-  http://www.jacco2.dds.nl/networking/openswan-l2tp.html
+.TH xl2tpd 8 "Sep 2020"
+
+.SH NAME
+xl2tpd \- Layer 2 Tunnelling Protocol Daemon.
+
+.SH DESCRIPTION
+xl2tpd is an implementation of the Layer 2 Tunnelling Protocol (RFC 2661).
+L2TP allows to tunnel PPP over UDP. Some ISPs use L2TP to tunnel user sessions
+from dial-in servers (modem banks, ADSL DSLAMs) to back-end PPP servers.
+Another important application is Virtual Private Networks (VPN) where the
+IPsec protocol is used to secure the L2TP connection (L2TP/IPsec, RFC 3193).
 
-Currently maintained by Xelerance
-  http://www.xelerance.com/software/xl2tpd/
+xl2tpd works by opening a pseudo-tty for communicating with pppd.
+It runs completely in userspace but supports kernel mode L2TP.
 
-xl2tpd implements the Layer 2 Tunnelling Protocol, defined by RFC 2661.
+xl2tpd supports IPsec SA Reference tracking to enable overlapping internak
+NAT'ed IP's by different clients (eg all clients connecting from their
+linksys internal IP 192.168.1.101) as well as multiple clients behind
+the same NAT router.
+
+On Linux xl2tpd supports the pppol2tp kernel mode operations by using
+kernel 2.6.23 or higher, or via a patch in contrib for 2.4.x kernels.
+Note that kernel mode and IPsec SA Reference tracking do not yet work
+together.
+
+This implementation is based on L2TPd 0.61 from http://www.marko.net/l2tp
+and patches collected by Jacco de Leeuw at
+http://www.jacco2.dds.nl/networking/openswan-l2tp.html.
 
-.SH "OPTIONS"
+.SH OPTIONS
 .TP 
 .B -D
-This option prevents xl2tpd from detaching from the terminal and
-daemonizing.  
+This option prevents xl2tpd from detaching from the terminal and daemonizing.
 
 .TP 
 .B -l
 This option tells xl2tpd to use syslog for logging even when \fB\-D\fR
 was specified.
 
 .TP
-.B -c <config file>
-Tells xl2tpd to use an alternate config file.  Default is
-/etc/xl2tpd/xl2tpd.conf. Fallback configuration file is
```

Comment:
I thiunk it would still be helpful to specify the default is /etc/xl2tpd/xl2tpd.conf.